### ISSUE TITLE
Enhancement Add http command which starts new effect and disables old ones

### ISSUE THREAD

**Issue by psander247** [FalconChristmas/fpp#582]
It wouldbe great if there could be a command which starts a new effect and stops the older ones after that. 
Currently if you start a new effect the old ones stay active... maybe there could be a problem if there are too  much playing at once.
I tried stopping the effect with id 0 before startinga new one,but then you have a hickup at your lights.

TIA

**Comment by patdelaney** [comment]
just taking a stab at this but can you script it and then use a curl
command?

Pat

**Comment by darylc** [comment]
This looks to be a duplicate of #735 from the same user
